components/stub-generation/community.tsx

```diff
@@ -12,7 +12,7 @@ const SocialCloud: React.FC = () => {
           🐰 Join the Keploy community ✨
         </h2>
       </div>
-      <div className="social-cloud" >
+      <div className="social-cloud gap-2" >
```

Comment:
for small screen you can keep it as it is

Reply:
okay, Thanks!